### PR TITLE
fix: fallback to english for knowledge panels

### PR DESCRIPTION
**Pull request by alexgarel**
fixes #6771

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -407,7 +407,7 @@ sub process_template($$$) {
 
 	# Display one taxonomy entry in the target language
 	$template_data_ref->{display_taxonomy_tag} = sub ($$) {
-		return display_taxonomy_tag($lc, $_[0], $_[1]);
+		return display_taxonomy_tag_name($lc, $_[0], $_[1]);
```
Comment (stephanegigandet) [inline lib/ProductOpener/Display.pm:410]: This is ok I guess, I don't think we have templates where we do want the tag with the leading language code. We might in the future though, e.g. if we want to use panels to link to a list of products with a specific tag.

Another option is to change display_taxonomy_tag to display_taxonomy_tag_name in all templates.

**Review by stephanegigandet — APPROVED** [review]

File: lib/ProductOpener/Display.pm (hunk 1 above)
Comment (alexgarel) [reply lib/ProductOpener/Display.pm:410]: I will change to have a specific display_taxonomy_tag_name available in templates

Comment (alexgarel) [reply lib/ProductOpener/Display.pm:415]: I've change that, to instead have two function and use appropriate one in knowledge panels.

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

**Pull request merged**